src/commands/schema/push.mjs

```diff
@@ -1,97 +1,9 @@
 //@ts-check
 
-import path from "path";
-
-import { container } from "../../cli.mjs";
 import { yargsWithCommonQueryOptions } from "../../lib/command-helpers.mjs";
-import { ValidationError } from "../../lib/errors.mjs";
-import { getSecret } from "../../lib/fauna-client.mjs";
-import { reformatFSL } from "../../lib/schema.mjs";
+import { pushSchema } from "../../lib/schema.mjs";
```

Comment:
why move it into the `schema.mjs`?

Reply:
I moved it back into commands. The existing convention is to keep commands in commands, and import them directly if another command uses the command wholesale.